keyboards/splitkb/aurora/sweep/keymaps/agrski/readme.md

```diff
@@ -0,0 +1,35 @@
+# Aurora Sweep's Default Keymap
+
+A chord-based keymap for effective typing
+===
+
+While the default keymap makes heavy use of held keys as layer switches, this keymap instead prioritises chords.
+It uses set keys to navigate up and down layers and emphasises the thumb keys as modifiers.
+The idea is to avoid having to hold keys for extended periods or have too many layers to remember.
+
+It makes the _super_ modifier always available for use with i3wm or equivalent window managers.
+
+Layer map
```

Comment:
:wrench: These need to be updated to reflected the current state

Reply:
Fixed in https://github.com/agrski/qmk_firmware/pull/1/commits/2d0c789b969da4ef2c1a47992b2b6aaaab20db89